frappe/utils/make_random.py

```diff
@@ -36,8 +36,13 @@ def get_random(doctype, filters=None, doc=False):
 	else:
 		condition = ""
 
-	out = frappe.db.sql("""select name from `tab%s` %s
-		order by RAND() limit 0,1""" % (doctype, condition))
+	out = frappe.db.multisql({
```

Comment:
This query dynamically builds condition and is not trivial to replace - ok to leave as-is?

Reply:
Same as above. Criterion.any does not appear to work when an entire string is supplied - only when a pypika field is supplied with some constant value. E.g. this is supported Criterion.any(table[fieldname] == 'value') but this is not: Criterion.any(""" 'fieldname' = 'value' """)

Therefore I don't really see a way to refactor this locally. Please advise if qb can be used here.

Reply:
IMO we can leave queries like these out as long as they are covered 😅